test/validation/api/ipsec/ipsec.c

```diff
@@ -309,6 +306,11 @@ void ipsec_sa_param_fill(odp_ipsec_sa_param_t *param,
 
 	if (auth_key_extra)
 		param->crypto.auth_key_extra = *auth_key_extra;
+
+	param->lifetime.soft_limit.bytes = 900 * 1000;
+	param->lifetime.hard_limit.bytes = 1000 * 1000;
+	param->lifetime.soft_limit.packets = 9000 * 1000;
+	param->lifetime.hard_limit.packets = 10000 * 1000;
```

Comment:
Are these limits exceeded in any test?

Reply:
Not even close. I just wanted to put nonzero values there to force the implementation to perform lifetime checking just to get more of the code run, even if not really tested, in the tests. Real life time tests seem to be missing.

The limits are largish (compared to the handful of packets the tests actually send) to prevent us getting
an unexpected expiration indication if an implementation (like the linux-gen reference implementation) optimizes lifetime checking in ways that make it somewhat inexact.

Reply:
Some code comment might be good here because the numbers look like they might have some meaning, but they're really just large values.